src/Common/Bridge/BridgeMethodsRule.php

```diff
@@ -41,7 +41,12 @@ public function apply(AbstractNode $node)
             return;
         }
 
-        $firstInterface = $node->getInterfaces()[0];
+        $firstInterface = $this->findFirstClassInterface($node);
```

Comment:
This will exclude the bridges without interfaces from executing. We should replace this with other rule or set a new violation here, but not silent quit I guess.

Reply:
A new violating sounds fine to me. With a specific text for this error (missing).